### DISCUSSION TITLE
Age from Date

### DISCUSSION THREAD

**Discussion by JamCam84** [Budibase/budibase#3945] [Help]
I have a form with date of birth, how can I dynamically calculate the age where it keeps on updating

**Comment by mjashanks** [comment]
You should be able to do this using a Javascript binding. This article on stack overflow describes a method to calculate age from date of birth: https://stackoverflow.com/questions/4060004/calculate-age-given-the-birth-date-in-the-format-yyyymmdd

Your binding will look like

```javascript
function getAge(dateString) {
   .... 
}

return getAge($("Your.Field.From.Bindings"));
```
